compiler/testData/diagnostics/tests/typealias/typeAliasForProjectionInSuperInterfaces.kt

```diff
@@ -22,11 +22,11 @@ class Test4 : InvT<<!PROJECTION_IN_IMMEDIATE_ARGUMENT_TO_SUPERTYPE!>*<!>>
 class Test5 : InvT<InvT<*>>
 
 class Test6 : <!EXPANDED_TYPE_CANNOT_BE_INHERITED!>OutStar<!>
-class Test7 : <!EXPANDED_TYPE_CANNOT_BE_INHERITED!>OutOut<!>
+class Test7 : OutOut
 class Test8 : OutT<Int>
 class Test9 : OutT<<!PROJECTION_IN_IMMEDIATE_ARGUMENT_TO_SUPERTYPE!>out<!> Int>
 
 class Test10 : <!EXPANDED_TYPE_CANNOT_BE_INHERITED!>InStar<!>
-class Test11 : <!EXPANDED_TYPE_CANNOT_BE_INHERITED!>InIn<!>
+class Test11 : InIn
```

Comment:
Changes like these make me uncomfortable with this change. I don't feel like I understand what is going on here. It looks like all the actual functional tests are passing, but it is clear that this changes things in subtle ways.